Description: Turn Java convenience test into an expected failure
 http://gcc.gnu.org/bugzilla/show_bug.cgi?id=42143 means that there are
 always duplicate .dummy symbols, so the link fails.
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no
Last-Update: 2011-02-16

Index: b/tests/convenience.at
===================================================================
--- a/tests/convenience.at
+++ b/tests/convenience.at
@@ -272,6 +272,8 @@
   $LIBTOOL --tag=GCJ --mode=compile $GCJ $GCJFLAGS -c A$i.java
   $LIBTOOL --tag=GCJ --mode=link $GCJ $GCJFLAGS $LDFLAGS -o liba$i.la A$i.lo
 done
+# http://gcc.gnu.org/bugzilla/show_bug.cgi?id=42143
+AT_XFAIL_IF([:])
 AT_CHECK([$LIBTOOL --tag=GCJ --mode=link $GCJ $GCJFLAGS $LDFLAGS -o liba12.la liba1.la liba2.la -rpath /notexist],
 	 [0],[ignore],[ignore])
 AT_CHECK([$LIBTOOL --tag=GCJ --mode=link $GCJ $GCJFLAGS $LDFLAGS -o liba123.la A3.lo liba1.la liba2.la -rpath /notexist],
